source/features/conversation-activity-filter.tsx

```diff
@@ -37,38 +38,24 @@ function isWholeReviewEssentiallyResolved(review: HTMLElement): boolean {
 }
 
 function processSimpleComment(item: HTMLElement): void {
-	if (currentSetting === 'hideEvents') {
-		return;
-	}
-
 	// Hide comments marked as resolved/hidden
 	if (select.exists('.minimized-comment > details', item)) {
-		item.classList.add(hiddenClassName);
+		item.classList.add(collapsedClassName);
 	}
 }
 
 function processReview(review: HTMLElement): void {
-	if (currentSetting === 'hideEvents') {
-		return;
-	}
-
 	if (isWholeReviewEssentiallyResolved(review)) {
-		review.classList.add(hiddenClassName);
+		review.classList.add(collapsedClassName);
 		return;
 	}
 
 	for (const threadContainer of select.all('.js-resolvable-timeline-thread-container[data-resolved="true"]', review)) {
-		threadContainer.classList.add(hiddenClassName);
+		threadContainer.classList.add(collapsedClassName);
 	}
 }
 
-function processPage(): void {
-	removeClassFromAll(hiddenClassName);
-
-	if (currentSetting === 'default') {
-		return;
-	}
-
+const processPage = onetime(() => {
```

Comment:
I don't think we want `onetime` here, because it won't be called again when a new comment appears.